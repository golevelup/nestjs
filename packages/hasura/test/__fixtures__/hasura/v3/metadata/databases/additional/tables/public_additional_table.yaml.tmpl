table:
  name: additional_table
  schema: public
