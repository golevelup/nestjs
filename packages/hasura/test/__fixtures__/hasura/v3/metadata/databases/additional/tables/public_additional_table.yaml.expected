table:
  name: additional_table
  schema: public
event_triggers:
  - name: additional_table_event_handler
    definition:
      enable_manual: false
      delete:
        columns: '*'
    retry_conf:
      num_retries: 3
      interval_sec: 15
      timeout_sec: 100
    webhook_from_env: NESTJS_EVENT_WEBHOOK_ENDPOINT
    headers:
      - name: NESTJS_SECRET_HEADER
        value_from_env: NESTJS_WEBHOOK_SECRET_HEADER_VALUE
